js/header_dialog.js

```diff
@@ -225,54 +209,133 @@ function HeaderDialog(dialog, onSave) {
+                                                break;
+                                        case 3:
+                                                if(data[i]!=null) {
+                                                        $(this).val(data[i].toFixed(0));
+                                                        $(this).attr('decPl', 2);
+                                                        $(this).removeClass('missing');
+                                                        console.log($(this).val(data[i].toFixed(0)));
+                                                } else {
+                                                        $(this).val('');
+                                                        $(this).addClass('missing');                                                        }
+                                                i++;
+                                                break;
+                                        case 4:
+                                                if(data[i]!=null) {
+                                                        $(this).val(data[i].toFixed(0));
+                                                        $(this).attr('decPl', 3);
+                                                        $(this).removeClass('missing');
+                                                        console.log($(this).val(data[i].toFixed(0)));
+                                                } else {
+                                                        $(this).val('');
+                                                        $(this).addClass('missing');
+                                                }
+                                                i++;
+                                                break;
+                                        case 5:
+                                                if(data[i]!=null) {
+                                                        $(this).val(data[i].toFixed(0));
+                                                        $(this).attr('decPl', 3);
+                                                        $(this).removeClass('missing');
+                                                        console.log($(this).val(data[i].toFixed(0)));
+                                                } else {
+                                                        $(this).val('');
+                                                        $(this).addClass('missing');
+                                                }
+                                                i++;
+                                                break;
+                                        }
+                                } else $(this).closest('tr').addClass('missing');
+            })
+        }
+
+        function populateBandwidth(name, data) {
+                let i = 0;
+        const nameElem = $('.pid_bandwidth .' + name + ' input');
+        nameElem.each(function () {
```

Comment:
The indentation looks weird here. I wonder what's going on... using tabs instaead of spaces maybe?